tools/edbconnect/main/edb-enclave.json

```diff
@@ -0,0 +1,19 @@
+{
+  "exe": "main",
+  "key": "testnet.pem",
+  "debug": true,
+  "heapSize": 4096,
```

Comment:
Remind me, does the enclave allocate the memory preemptively? If so this might be a bit huge

Reply:
Yeah good shout I think it might allocate greedily. I'll cut it in half and check it still works